### PR TITLE
Fixed eslint-config-airbnb

### PR DESCRIPTION
**Pull request by MiladMoulayi**
Added airbnb/hooks to .eslintrc.js. Doesn't complain about file name extensions or double-quotes anymore.

### CODE REVIEW

**Comment by MiladMoulayi** [comment]
paused for further fixes

**Comment by MiladMoulayi** [comment]
Reopened with corrections. Please review and approve at your convenience.

**Review by gocodezhang — APPROVED** [review]
Thanks for making the changes

**Comment by MiladMoulayi** [comment]
Added more fixes to .eslintrc and .babelrc files.

File: client/src/App.jsx [hunk 1]
```diff
@@ -1,16 +1,14 @@
...
+import { React, useEffect, useState } from 'react';
 import { ReactDom, render } from 'react-dom';
+import Overview from './components/overview/Overview';
 
+function App() {
+  const { productId, setProductId } = useState('');
+  const { outfitArr, setOutfitArr } = useState([]);
 
+  return (
+    <Overview />
```
Comment (HeyMaximus) [inline client/src/App.jsx:10]: Is the Overview component redundant? It seems like you intend to nest all our widget views in a separate component called Overview, before presenting it to App.  So architecture becomes x4 Widgets --> Overview --> App --> index. Or have I misunderstood and you're calling your widget "Overview"?

My original understanding is our architecture is x4 Widgets --> App --> index.

If your intention is x4 Widgets --> Overview --> App --> index,
what is the consideration/advantage you wanted to achieve with Overview?

Comment (MiladMoulayi) [reply client/src/App.jsx:10]: Overview is the Product Overview widget. I'm going to pass all the subcomponents to Overview, which will then be passed to App.

**Review by HeyMaximus — APPROVED** [review]

**Pull request merged**